src/components/Users/Users.js

```diff
@@ -47,8 +49,44 @@ const Users = (props) => {
       </div>
       <div>
         {user.followed
-          ? <button onClick={() => props.unfollow(user.id)}>Unfollow</button>
-          : <button onClick={() => props.follow(user.id)}>Follow</button>}
+
+          ? (
+            <button onClick={() => {
```

Comment:
Consider moving the onClick handler out of JSX.

Reply:
/fxd